src/components/ViolenceTestCards/index.jsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import { Col, Row } from "react-bootstrap";
+import { useTranslation } from "react-i18next";
+import "./index.scss";
+
+const ViolenceTestCards = () => {
+  const { t } = useTranslation();
+  const cardsArr = [...t("violenceTestPage.testSpec", { returnObjects: true })];
+  const checkboxItemsArr = [
+    [
+      ...t("violenceTestPage.testSpec.0.checkboxItems", {
+        returnObjects: true,
+      }),
+    ],
+    [
+      ...t("violenceTestPage.testSpec.1.checkboxItems", {
+        returnObjects: true,
+      }),
+    ],
+    [
+      ...t("violenceTestPage.testSpec.2.checkboxItems", {
+        returnObjects: true,
+      }),
+    ],
+  ];
+
+  const cards = cardsArr.map((card) => {
+    return (
+      <div className="col">
+        <h3>{t(card.category)}</h3>
+        <h5>{t(card.subtitle)}</h5>
+      </div>
+    );
+  });
+
+  const checkboxItems = checkboxItemsArr.map((items) => {
+    return (
+      <form className="test col">
+        {items.map((item) => (
```

Comment:
you still need the key, but what i meant is you were doing item.id but i don't see an ID in the JSON file for translation, so as far as i know it was doing nothing (or assigning it to undefined)

Reply:
Actually this variable is array, do I really need to add id to it?
I added id to the main objects.